__tests__/GraphBuilder.test.js

```diff
@@ -71,12 +83,22 @@ describe('GraphBuilder', () => {
 
       expect(graph.has(propertyTriple)).toBeTruthy()
 
-      let result = graph.filter(quad => quad.object.equals(rdf.literal('Very colorful')))
+      // Multiple items
+      const result1 = graph.filter(quad => quad.object.equals(rdf.literal('Very colorful'))).toArray()
 
-      expect(result.toArray().length).toEqual(1)
+      expect(result1.length).toEqual(1)
 
-      result = graph.filter(quad => quad.object.equals(rdf.literal('Sparkly')))
-      expect(result.toArray().length).toEqual(1)
+      const result2 = graph.filter(quad => quad.object.equals(rdf.literal('Sparkly'))).toArray()
+
+      expect(result2.length).toEqual(1)
+
+      // Multiple resources (notes)
+      const result3 = graph.filter(quad => quad.object.equals(rdf.literal('Shiney'))).toArray()
+
+      expect(result3.length).toEqual(1)
+      // Each note has separate blank nodes.
+      expect(result1[0].subject).toEqual(result2[0].subject)
```

Comment:
showing that these 2 things are equal ... isn't showing that they are separate blank nodes ... what is this line actually testing?

Reply:
Clarified.